File: src/TmrOverlay.App/Overlays/StreamChat/StreamChatOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.StreamChat;

internal static class StreamChatOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "stream-chat",
        DisplayName: "Stream Chat",
        DefaultWidth: 380,
        DefaultHeight: 520,
        ShowSessionFilters: false,
        ShowScaleControl: false,
        ShowOpacityControl: false);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/StreamChat/StreamChatOverlayDefinition.swift
import AppKit

enum StreamChatOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "stream-chat",
        displayName: "Stream Chat",
        defaultSize: NSSize(width: 380, height: 520),
        showSessionFilters: false,
        showScaleControl: false,
        showOpacityControl: false
    )
}
